docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -14,6 +14,25 @@ additional driver and software components are properly installed and running.
 
 [Intel SGX]: https://www.intel.com/content/www/us/en/architecture-and-technology/software-guard-extensions.html
 
+## BIOS Configuration
+To set up a TEE (Intel SGX), you need to configure BIOS. First, **update BIOS 
+to the latest version with the latest microcode**. Then proceed with BIOS 
+configuration. Note that not all settings may apply to your BIOS. In that case, 
+configure only the relevant ones. Please set the BIOS settings as follows:
+
+- **SGX**: ENABLE
+- **Hyperthreading**: DISABLE
+- **Intel SpeedStep**: DISABLE
+- **SecureBoot**: DISABLE
+- **All Internal Graphics**: DISABLE
+- **Turbo Mode**: DISABLE
+- **CPU AES**: ENABLE
+
+To test if your settings are correct, you may use the [attestation tool] for 
```

Comment:
Also link to the tool's binary release (e.g. in parenthesis).

Reply:
The link can simply be the tools release page https://github.com/oasisprotocol/tools/releases. I don't see why anyone would want to fetch and compile the tool manually.

Reply:
Added link to the tool's binary release in parenthesis. I've left the link to the readme because this is where the tool's output is explained. Hope this is ok.